make/cluster.sh

```diff
@@ -101,18 +101,18 @@ fi
 # hack/update-kind-images.sh.
 source "${here}/config/kind/kind_cluster_node_versions.env"
 case "$k8s_version" in
-1.18*) kind_image_sha=$KIND_IMAGE_SHA_K8S_118 ;;
-1.19*) kind_image_sha=$KIND_IMAGE_SHA_K8S_119 ;;
-1.20*) kind_image_sha=$KIND_IMAGE_SHA_K8S_120 ;;
-1.21*) kind_image_sha=$KIND_IMAGE_SHA_K8S_121 ;;
-1.22*) kind_image_sha=$KIND_IMAGE_SHA_K8S_122 ;;
-1.23*) kind_image_sha=$KIND_IMAGE_SHA_K8S_123 ;;
+1.18*) image="${kind_image_repo}:v1.18.0@$KIND_IMAGE_SHA_K8S_118" ;;
+1.19*) image="${kind_image_repo}:v1.19.0@$KIND_IMAGE_SHA_K8S_119" ;;
+1.20*) image="${kind_image_repo}:v1.20.0@$KIND_IMAGE_SHA_K8S_120" ;;
+1.21*) image="${kind_image_repo}:v1.21.0@$KIND_IMAGE_SHA_K8S_121" ;;
+1.22*) image="${kind_image_repo}:v1.22.0@$KIND_IMAGE_SHA_K8S_122" ;;
+1.23*) image="${kind_image_repo}:v1.23.0@$KIND_IMAGE_SHA_K8S_123" ;;
```

Comment:
Right, that's something I overlooked.

In my mind, `make/cluster.sh` is the source of truth for everything about kind. With that in mind, I added a new command `make/cluster.sh --update-images` so that images can be updated directly by the script itself. PTAL in b3680abdb00ef938ca73b20b376d218cfb8356fb, I also put the digests inside the script itself.